### ISSUE TITLE
Improve teams page UI

### ISSUE THREAD

**Issue by danieljames-dj** [thewca/worldcubeassociation.org#8164]
Currently, the teams page (https://www.worldcubeassociation.org/teams-committees) have all the members listed in single page. Instead can we have a left side bar where the team names will be listed and when we select a team, the members of just that team will be shown.

This will also give more area for each team so that maybe we can consider adding more metadata like since when the member is part of the team, etc.

What are your suggestions on having this UI improvement? Will it be better or is current UI more better?

**Comment by dunkOnIT** [comment]
When I went to the Teams page recently, I immediately thought that a design like your Version 1 delegates page would be far better - especially because I often want to jump to a team very far down the page. Strongly support this, and suggest we pursue it once your Delegates UI PR is merged as a reference.

**Comment by danieljames-dj** [comment]
Thanks @dunkOnIT. I just read your message (I don't remember whether I saw it earlier, maybe forgot to reply then).

Now since Delegates page is done, maybe we can implement Teams page in a similar way. But having Board & Officers along with teams & committees doesn't look good. I've created another issue to handle the Board & Officers page (https://github.com/thewca/worldcubeassociation.org/issues/8659).

**Comment by danieljames-dj** [comment]
This has been fixed, but there were lots of critics around this. The details of that can be found here: https://github.com/thewca/worldcubeassociation.org/issues/10093.
